tests/framework/extensions/charts/ranchergatekeeper.go

```diff
@@ -26,12 +26,22 @@ const (
 
 // InstallRancherGatekeeperChart installs the OPA gatekeeper chart
 func InstallRancherGatekeeperChart(client *rancher.Client, installOptions *InstallOptions) error {
-	hostWithProtocol := fmt.Sprintf("https:// %s", client.RancherConfig.Host)
```

Comment:
we no longer need to get the host with the protocol?

Reply:
Yes, that's correct. `server-url` v3 setting's value returns the host with the protocol (unless it's overwritten as another string).